packages/curve-ui-kit/src/widgets/Footer/Link.tsx

```diff
@@ -0,0 +1,47 @@
+import LinkMui from '@mui/material/Link'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { ReactNode } from 'react'
+import { SizesAndSpaces } from 'curve-ui-kit/src/themes/design/1_sizes_spaces'
+
+export type LinkProps = {
+  label: string
+  href: string
+  icon?: ReactNode
+}
+
+export const Link = ({ label, href, icon }: LinkProps) => (
+  <LinkMui
```

Comment:
I see your response in [this comment](https://github.com/curvefi/curve-frontend/pull/500#discussion_r1872260759) however, I think it should follow the design system (yeah that video seems correct according to that)

Reply:
As discussed in chat, will need tutti opinion. Can be done in diff PR.